src/test/forward_index_test.cpp

```diff
@@ -128,9 +128,25 @@ int forward_index_tests()
     num_failed += testing::run_test("forward-index-read-file-corpus", [&]()
     {
         ceeaus_forward_test();
-        system("rm -rf ceeaus-* test-config.toml");
     });
 
+    num_failed += testing::run_test("forward-index-build-uninvert", [&]()
+    {
+        system("rm -rf ceeaus-*");
+
+        // hack to inject "uninvert = true" at the top of the config file
+        auto cfg_contents = filesystem::file_text("test-config.toml");
+        cfg_contents = "uninvert = true\n" + cfg_contents;
+        filesystem::delete_file("test-config.toml");
```

Comment:
Ok, cool. I can also change my hack in search-app to use this feature as well then.